firecrown/models/cluster/abundance_data.py

```diff
@@ -63,7 +63,7 @@ def get_observed_data_and_indices_by_survey(
                 # pylint: disable=no-member
                 data_type = sacc.standard_types.cluster_mean_log_mass
             else:
-                raise NotImplementedError(cluster_property)
+                continue
```

Comment:
Here is another example from my last comment. I want it to go through the data and just do not do anything if it is not a property of the statiscs. However this should be replaced with another error if the provided properties are not the ones from the ClusterProperty module